src/Tools/CoreFx.Tools/project.json

```diff
@@ -20,10 +21,12 @@
         "portable-net45+win8+wpa81"
       ]
     },
-    "net45": {
+    "net46": {
       "dependencies": {
-        "Microsoft.TargetingPack.NETFramework.v4.5": "1.0.1"
+        "Microsoft.TargetingPack.NETFramework.v4.6": "1.0.1",
+        "System.Reflection.Metadata": "1.1.0",
```

Comment:
Sorry, what was the issue? I would think that the newer package versions would have all of the methods that this one does, unless we broke back-compat ?